doc/source/users-list.md

```diff
@@ -0,0 +1,9 @@
+# Zero to JupyterHub users
```

Comment:
I would change the title to Gallery to give more flexibility over time with content